packages/opencensus-core/src/trace/config/types.ts

```diff
@@ -37,6 +37,8 @@ export interface TracerConfig {
   logger?: Logger;
   /** A propagation instance */
   propagation?: Propagation;
+  /** Trace Parameters */
+  traceParameters?: TraceParameters;
```

Comment:
To me, `traceParameters` feels a bit generic, and it seems like all these params are related to size. What would you think about calling this something like `traceSizeParameters`, or if that's too long, `traceSizeConfig` or something similar?

Reply:
Renamed from `TraceParameters` to `TraceParams`